{{cookiecutter.github_repo_name}}/Dockerfile

```diff
@@ -4,9 +4,14 @@ LABEL maintainer={{cookiecutter.git_email}}
 # Switch to super user to perform privileged operatiosn
 USER root
 
+# Change this value to regenerate the whole image
+ENV _UPDATED 2022-04-08
```

Comment:
Cosa cambia tra la variazione di questo valore e l'uso di docker con `--no-cache`?

Reply:
@OpenCode se modifichi il Dockerfile, Docker smette di usare i layer da quel punto in poi. L'effetto finale è più o meno identico a --no-cache, ma non hai bisogno di passare il parametro